rpc/wrtc_base_channel.go

```diff
@@ -109,7 +109,7 @@ func newBaseChannel(
 				//
 				// We chose here to call close for all cases of `Disconnected`, `Failed` and
 				// `Closed`. We rely on pion's `PeerConnection.Close` method being idempotent.
```

Comment:
```suggestion
				// `Closed`. We rely on pion's `PeerConnection.GracefulClose` method being idempotent.
				// We "gracefully" close to wait for the read loop to complete.
```
At least one mention of the `Graceful` usage might be helpful.